<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LiveRoom</class>
 <widget class="QWidget" name="LiveRoom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>842</width>
    <height>676</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_roomid">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_allusername">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>20</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="text">
    <string>所有用户</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_allusername">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>90</y>
     <width>171</width>
     <height>391</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>21</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_camera">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>521</width>
     <height>381</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_talkinformation">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>570</y>
     <width>361</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>21</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_sendtalk_information">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>570</y>
     <width>161</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_startliving">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>510</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="text">
    <string>开启直播</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_closeliving">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>590</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="text">
    <string>关闭直播</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
